cloudify_aws/common/__init__.py

```diff
@@ -116,3 +116,7 @@ def verify_resource_exists(self):
         if not getattr(self, 'properties', {}):
             return False
         return True
+
```

Comment:
Can we add more to the docstring so that future devs don't need to ask what this is for? Also in the decorator.